docs/source/tutorials/quickstart.rst

```diff
@@ -2,145 +2,150 @@
+
+If you get an error
+message instead, you do not see a video, or if the video output does not
+look like the preceding animation, it is likely that Manim has not been
+installed correctly. Please refer to the :doc:`../installation/troubleshooting`
+page for more information.
 
 
 ***********
 Explanation
 ***********
 
-Let's go line by line over the script we just executed to see how manim was
-able to generate the video.
+Let's go over the script you just executed line by line to see how Manim was
+able to draw the circle.
 
-The first line
+The first line imports all of the contents of the library:
 
 .. code-block:: python
 
    from manim import *
 
-imports all of the contents of the library.  This is the recommended way of
-using manim, as usually in a single script you will be using quite a few names
-from the manim namespace.  In particular, this line includes all of the names
-used in the script: ``Scene``, ``Circle``, ``PINK`` and ``Create``.
+This is the recommended way of using Manim, as a single script often uses
+multiple names from the Manim namespace. In your script, you imported and used
+``Scene``, ``Circle``, ``PINK`` and ``Create``.
 
-Now let's look at the next two lines.
+Now let's look at the next two lines:
 
 .. code-block:: python
 
-   class SquareToCircle(Scene):
+   class CreateCircle(Scene):
        def construct(self):
            ...
 
-Most of the time, the code for scripting an animation with manim will go inside
-the :meth:`~.Scene.construct` method of a class that derives from :class:`.Scene`.  Inside this
-method, you will create objects, display them on screen, and animate them.
+Most of the time, the code for scripting an animation is entirely contained within
+the :meth:`~.Scene.construct` method of a :class:`.Scene` class.
+Inside :meth:`~.Scene.construct`, you can create objects, display them on screen, and animate them.
 
-The next two lines create a circle and set its color and opacity.
+The next two lines create a circle and set its color and opacity:
 
 .. code-block:: python
 
            circle = Circle()  # create a circle
            circle.set_fill(PINK, opacity=0.5)  # set the color and transparency
 
 Finally, the last line uses the animation :class:`.Create` to display the
-circle on the screen.
+circle on your screen:
 
 .. code-block:: python
 
            self.play(Create(circle))  # show the circle on screen
 
-.. tip:: Every animation must be contained within the :meth:`~.Scene.construct` method of a
-         class that derives from :class:`.Scene`.  Other code, for example auxiliary
+.. tip:: All animations must reside within the :meth:`~.Scene.construct` method of a
+         class derived from :class:`.Scene`.  Other code, such as auxiliary
          or mathematical functions, may reside outside the class.
 
 
-Some bells and whistles
-***********************
+Transforming a square into a circle
+*************************************************
```

Comment:
Minor point, but are all these asterisks needed? It just has to be the same length as the section title right?

Reply:
To be honest, I didn't realize that was the reason the asterisk breaks were differing lengths so I just copied from the longest one and went with it. Fixed.